pom.xml

```diff
@@ -135,8 +135,8 @@
 
         <netty-tcnative-version>2.0.39.Final</netty-tcnative-version>
         <!-- We normally get this from common, but Vertx is built against this -->
-        <netty.version>4.1.69.Final</netty.version>
-        <netty-codec-http2-version>4.1.69.Final</netty-codec-http2-version>
+        <netty.version>4.1.72.Final</netty.version>
+        <netty-codec-http2-version>4.1.72.Final</netty-codec-http2-version>
```

Comment:
Netty version 4.1.73 appears to be available and recommended: https://netty.io/downloads.html

I imagine there's some coordination among repos, so if .72 is what's needed, that's fine!

Reply:
yup - we've had history where a vertx/netty mismatch causes very tough to debug problems so we've just been playing it safe and keeping netty version to whatever vertx uses

Reply:
+1.  Yeah, I've learned that to avoid mismatching Netty versions the hard way.:)